luthier/config/src/config.lua

```diff
@@ -36,16 +37,34 @@ local metricConfig = metricsUtils.build_metric_config(metrics)
 local metricMakerConfig = metricMakerUtils.build_metric_maker_config(metricMakers)
 local physicalTableConfig, logicalTableConfig = tableUtils.build_table_config(tables)
 
+--- write to testResource/tableName
+local function write_table(targetDir, tableName, configTable)
```

Comment:
I don't think we should be using "table" to refer to the the things we're writing. Too
easy to confuse with "logical table." Rather, I think we should use something like `config_entity`. So,
this function would be called `write_config_entity`. The `tableNames` variable below would
be `configEntityNames`, and so on.

Reply:
agreed.